DSharpPlus/Entities/DiscordChannel.cs

```diff
@@ -203,6 +204,7 @@ public Task<DiscordMessage> SendMessageAsync(string content = null, bool tts = f
         /// <exception cref="Exceptions.NotFoundException">Thrown when the channel does not exist.</exception>
         /// <exception cref="Exceptions.BadRequestException">Thrown when an invalid parameter was provided.</exception>
         /// <exception cref="Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
+        [Obsolete("Please use the new DiscordMessage Builder.  This method will be removed in a future version", false)]
```

Comment:
There's one extra space.

Reply:
This is no longer applicable as these methods were tossed.